src/sentry/api/endpoints/api_tokens.py

```diff
@@ -35,23 +37,36 @@ class ApiTokensEndpoint(Endpoint):
     authentication_classes = (SessionNoAuthTokenAuthentication,)
     permission_classes = (IsAuthenticated,)
 
-    @method_decorator(never_cache)
-    def get(self, request: Request) -> Response:
+    @classmethod
+    def _get_appropriate_user_id(cls, request: Request) -> str:
+        """
+        Gets the user id to use for the request, based on what the current state of the request is.
+        If the request is made by a superuser, then they are allowed to act on behalf of other user's data.
+        Therefore, when GET or DELETE endpoints are invoked by the superuser, we may utilize a provided user_id.
+
+        The user_id to use comes from the GET or BODY parameter based on the request type.
+        For GET endpoints, the GET dict is used.
+        For all others, the DATA dict is used.
```

Comment:
@cathteng reference as to why the the params are like this. For POST requests, it's in the body as usual, and for GET it's in the query params, and so Django packages them accordingly